pkg/osquery/tables/wifi_networks/internal/assets/get-networks.ps1

```diff
@@ -0,0 +1,34 @@
+function Get-Networks {
+    function Convert-ByteArrayToString {
+        [CmdletBinding()] Param (
+            [Parameter(Mandatory = $True, ValueFromPipeline = $True)] [System.Byte[]] $ByteArray
+            )
+
+        $Encoding  = New-Object System.Text.ASCIIEncoding
+        $Encoding.GetString($ByteArray)
+    }
+
+    Add-Type -Path ".\nativewificode.cs"
```

Comment:
Interestingly, this works as written through this table. But if I try to run this from the command line it needs no quotes

Reply:
are you just execing the command from a powershell console? I've been testing the script by running `powershell.exe .\get-networks.ps1` from a **cmd.exe** session, and it works with the quoting.

Reply:
Yes. From inside a powershell script. But... my use case is boring, so it's hard to care.